keyboards/duck/tcv3/rules.mk

```diff
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```

Comment:
```suggestion
```

Reply:
?

Reply:
@tzarc in general, we like leaving the bootloader info in the rules.mk file, since on a lot of boards you can change the bootloader. Especially, if it's pro micro based.